tests/other/metadce/hello_libcxx_fastcomp_O2_EMULATED_FUNCTION_POINTERS.funcs

```diff
@@ -426,6 +433,7 @@ $___uflow
 $___uselocale
 $___vfprintf_internal
 $___wasi_syscall_ret
+$_abort_message
```

Comment:
this, and the other additions here, look bad to me - I think we should investigate if there isn't something we can ifdef away in this new code. OTOH I see the code size didn't need to change, so it's within a small range, but still.

Reply:
I tried to dig into these but didn't get very far.   It just looks like a normal amount of churn to me.   If its not a large code size increase I'm tempted to let it go.   But I will try to dig in a little more.

Reply:
Might be good to at least measure the code size regression here in a precise way.